### ISSUE TITLE
Error when MainWindow is minimized in OS X Dock

### ISSUE THREAD

**Issue by cbourgois** [sindresorhus/caprine#16]
When the main Window is minimized in OS X Dock, _Caprine > Préférences_ or _File > New Conversation_ produces an error window:

```
A JavaScript error occurred in the main process
Uncaught Exception:
TypeError: Cannot read property 'webContents' of undefined
    at sendAction (/path/to/Caprine.app/Contents/Resources/app.asar/menu.js:9:34)
    at click (/path/to/Caprine.app/Contents/Resources/app.asar/menu.js:27:6)
    at MenuItem.click (/path/to/Caprine.app/Contents/Resources/atom.asar/browser/api/lib/menu-item.js:58:20)
    at Function.delegate.executeCommand (/path/to/Caprine.app/Contents/Resources/atom.asar/browser/api/lib/menu.js:127:69)
```

**Comment by sindresorhus** [comment]
Fixed in 0.1.2.

**Comment by cbourgois** [comment]
Thanks!
